bl1/aarch32/bl1_context_mgmt.c

```diff
@@ -50,7 +50,7 @@ void *cm_get_context(uint32_t security_state)
 	return &bl1_cpu_context[security_state];
 }
 
-void cm_set_next_context(void *cpu_context)
+static void cm_set_next_context(void *cpu_context)
```

Comment:
I think the prototype was missed out and it is better to add a prototype in context_mgmt.h.